app/lib/common/widgets/attachments/attachment_item.dart

```diff
@@ -1,16 +1,19 @@
-import 'package:acter_flutter_sdk/acter_flutter_sdk_ffi.dart'
-    show Attachment, FfiBufferUint8;
-import 'package:atlas_icons/atlas_icons.dart';
+import 'package:acter/common/widgets/attachments/attachment_container.dart';
+import 'package:acter/common/widgets/attachments/views/file_view.dart';
+import 'package:acter/common/widgets/attachments/views/image_view.dart';
+import 'package:acter/common/widgets/attachments/views/video_view.dart';
+import 'package:acter_flutter_sdk/acter_flutter_sdk_ffi.dart' show Attachment;
 import 'package:flutter/material.dart';
 import 'package:flutter_riverpod/flutter_riverpod.dart';
-import 'package:flutter_gen/gen_l10n/l10n.dart';
 
 // Attachment item UI
 class AttachmentItem extends ConsumerWidget {
   final Attachment attachment;
+  final bool? openView;
```

Comment:
What is the use-case of `openView`?

Reply:
Should've add docs here. ```openView``` determines whether the pin attachment is clickable to be viewed in full screen or not. It's useful for pin lists where main attachment is shown and we don't want to have this opened unintentionally while opening pin list item.